hieradata_aws/class/staging/db_admin.yaml

```diff
@@ -29,3 +29,33 @@ govuk_env_sync::tasks:
     temppath: "/tmp"
     url: "govuk-staging-database-backups"
     path: "mysql"
+  "pull_pg_content_publisher_production_daily":
```

Comment:
Not sure why this is `pull_pg_...` as we specify the database in the YAML below...

Reply:
We can optimize this inconsistency when we've tested that all these jobs work.